api-participant/src/main/java/bio/terra/pearl/api/participant/service/file/ParticipantFileExtService.java

```diff
@@ -0,0 +1,100 @@
+package bio.terra.pearl.api.participant.service.file;
+
+import bio.terra.pearl.api.participant.service.AuthUtilService;
+import bio.terra.pearl.core.model.file.ParticipantFile;
+import bio.terra.pearl.core.model.participant.Enrollee;
+import bio.terra.pearl.core.model.participant.ParticipantUser;
+import bio.terra.pearl.core.service.exception.NotFoundException;
+import bio.terra.pearl.core.service.file.ParticipantFileService;
+import bio.terra.pearl.core.service.file.backends.FileStorageBackend;
+import bio.terra.pearl.core.service.file.backends.FileStorageBackendProvider;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Set;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+@Service
+@Slf4j
+public class ParticipantFileExtService {
+
+  private final ParticipantFileService participantFileService;
+  private final AuthUtilService authUtilService;
+  private final FileStorageBackend fileStorageBackend;
+
+  public ParticipantFileExtService(
+      ParticipantFileService participantFileService,
+      AuthUtilService authUtilService,
+      FileStorageBackendProvider fileStorageBackendProvider) {
+    this.participantFileService = participantFileService;
+    this.authUtilService = authUtilService;
+    this.fileStorageBackend = fileStorageBackendProvider.get();
+  }
+
+  public ParticipantFile get(
+      ParticipantUser participantUser, String enrolleeShortcode, String fileName) {
+    Enrollee enrollee =
+        authUtilService.authParticipantUserToEnrollee(participantUser.getId(), enrolleeShortcode);
```

Comment:
We don't have permissions on the PArticipantApi, but we probably should eventually move to annotation-based auth like the admin tool so logic like this gets more foolproof and easier to test.  I'll make a ticket